File: src/main/java/fi/aalto/cs/apluscourses/intellij/actions/ResetSettingsAction.java
package fi.aalto.cs.apluscourses.intellij.actions;

import com.intellij.openapi.actionSystem.AnActionEvent;
import com.intellij.openapi.project.DumbAwareAction;
import fi.aalto.cs.apluscourses.intellij.services.PluginSettings;
import fi.aalto.cs.apluscourses.utils.APlusLogger;
import org.jetbrains.annotations.NotNull;
import org.slf4j.Logger;

public class ResetSettingsAction extends DumbAwareAction {

  private static final Logger logger = APlusLogger.logger;

  @Override
  public void actionPerformed(@NotNull AnActionEvent e) {
    PluginSettings.getInstance().resetLocalSettings();
    logger.info("Reset local settings");
  }
}

File: src/main/kotlin/fi/aalto/cs/apluscourses/actions/ResetSettingsAction.kt
package fi.aalto.cs.apluscourses.actions

import com.intellij.openapi.actionSystem.AnActionEvent
import com.intellij.openapi.project.DumbAwareAction
import fi.aalto.cs.apluscourses.utils.APlusLogger

class ResetSettingsAction : DumbAwareAction() {
    override fun actionPerformed(e: AnActionEvent) {
//    PluginSettings.getInstance().resetLocalSettings();
        logger.info("Reset local settings")
    }

    companion object {
        private val logger = APlusLogger.logger
    }
}
